xrdsst/controllers/local_group.py

```diff
@@ -0,0 +1,319 @@
+from cement import ex
```

Comment:
This file shows 12 PEP8 warnings (your IDE should probably show these on the right as yellow markers)